example/benchmark.yml

```diff
@@ -70,10 +70,21 @@ plan:
   - name: Fetch some users from CSV (alternate syntax)
     request:
       url: /api/users/contacts/{{ item.id }}
-    with_items_from_csv: 
+    with_items_from_csv:
       file_name: ./fixtures/users.csv
       quote_char: "\'"
 
+  - name: POST some crafted JSONs stored in CSV
+    request:
+      url: /api/transactions
+      method: POST
+      body: '{{ item.txn }}'
```

Comment:
The body needs to be quoted. If not the interpolation is colliding with the yaml syntax